apps/condo/domains/billing/access/BillingMeterResource.js

```diff
@@ -6,33 +6,39 @@ const { get } = require('lodash')
 const { getById } = require('@core/keystone/schema')
 const { checkBillingIntegrationAccessRight } = require('@condo/domains/billing/utils/accessSchema')
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { USER_SCHEMA_NAME } = require('@condo/domains/common/constants/utils')
 
-async function canReadBillingMeterResources ({ authentication: { item: user } }) {
-    if (!user) return throwAuthenticationError()
+async function canReadBillingMeterResources ({ authentication: { item, listKey } }) {
+    if (!listKey || !item) return throwAuthenticationError()
+    if (item.deletedAt) return false
     return {}
 }
 
-async function canManageBillingMeterResources ({ authentication: { item: user }, originalInput, operation, itemId  }) {
-    if (!user) return throwAuthenticationError()
-    if (user.isAdmin || user.isSupport) return true
-    let contextId
-    if (operation === 'create') {
-        // NOTE: can create only by the integration account
-        contextId = get(originalInput, ['context', 'connect', 'id'])
-    } else if (operation === 'update') {
-        // NOTE: can update only by the integration account
-        if (!itemId) return false
-        const log = await getById('BillingIntegrationLog', itemId)
-        if (!log) return false
-        contextId = log.context
-    } else {
-        return false
+async function canManageBillingMeterResources ({ authentication: { item, listKey }, originalInput, operation, itemId  }) {
```

Comment:
this way it would be more readable, here and so on.

empty row before/after const/let/if/for/return/etc. blocks makes code more readable.

```
async function canManageBillingMeterResources ({ authentication: { item, listKey }, originalInput, operation, itemId  }) {
    if (!listKey || !item) return throwAuthenticationError()
    if (item.deletedAt) return false
    if (listKey === USER_SCHEMA_NAME) {
        let contextId

        if (item.isSupport || item.isAdmin) return true
        if (operation === 'create') {
            // NOTE: can create only by the integration account
            contextId = get(originalInput, ['context', 'connect', 'id'])
        } else if (operation === 'update') {
            // NOTE: can update only by the integration account
            if (!itemId) return false

            const log = await getById('BillingIntegrationLog', itemId)

            if (!log) return false

            contextId = log.context
        } else {
            return false
        }
        if (!contextId) return false

        const context = await getById('BillingIntegrationOrganizationContext', contextId)

        if (!context) return false

        const { integration: integrationId } = context

        return await checkBillingIntegrationAccessRight(item.id, integrationId)
    }
```